Add test instruction to dynamic pages

peiying2 opened the pull request:
- Need review and verification of the steps
- Line #85, this page only exists on staging. Is there a production link?
- Line #89, can't seem to find a way to get to the Sync content. Are the messages for first time sync setup?
- Line #91, for this feature, I can’t get to this message as the feature is not offered in the current product: It’s easy to turn off Tracking Protection for the website you’re on by clicking “Disable protection for this site.”

peiying2 committed:
Add test instruction to dynamic pages

- Need review and verify the steps
- Line #85, this page only exists on staging. Is there a production link?
- Line #89, can't seem to find a way to get to the Sync content. Are the messages for first time sync setup?
- Line #91, for this feature, I can’t get to this message as the feature is not offered in the current product: It’s easy to turn off Tracking Protection for the website you’re on by clicking “Disable protection for this site.”

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
 
 When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects.
+
+## Testing Dynamic Pages
```
No title case

peiying2 replied:
The whole doc is in this format.  Updating all titles except for the title of the document.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
 When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects.
+
+## Testing Dynamic Pages
+This section focuses on instructions for testing pages with dynamically generated content. Each page or topic is different in terms of steps or flow. These instructions could change overtime to reflect product design updates. Linguistic testing is the main focus. The instrutions below are detailed steps to get to the localized content so it can be reviewed in context.
```
overtime -> over time

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+* Once page is loaded, click on the link called **Download Firefox for another platform**
+* For alternative message, change URL to [Scene2](https://www.mozilla.org/en-US/firefox/new/?scene=2)
+* To fully review the Firefox download page messages, tests must go through following scenarios:
+.. * Desktop: Windows, macOS, Linux
```
Use a tab, not .. Checkout out of the page looks like
https://github.com/mozilla-l10n/documentation/blob/933bda12d046a80b2fc255bd0a02ffce7b2d381e/products/mozilla_org/test_mozilla_org.md

peiying2 replied:
fixed.

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+### [firefox/sync.lang](https://www.mozilla.org/en-US/firefox/sync/) 
+
+### firefox/tracking-protection-tour.lang
+* Start a new browser by clicking on the Firefox icon, select New Private Window on the menu list
```
You can simplify here: "Open Firefox and start a New private window"

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+* Text fits inside the box
+ 
+### [mozorg/about/manifesto.lang](https://www.mozilla.org/en-US/about/manifesto/)
+You can check out the manifestos without going to the slide mode. However, only in the slide mode, all the localized content can be reviewed.
```
the manifestos -> manifesto's principles?

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+ 
+### [mozorg/about/manifesto.lang](https://www.mozilla.org/en-US/about/manifesto/)
+You can check out the manifestos without going to the slide mode. However, only in the slide mode, all the localized content can be reviewed.
+* Click on the first manifesto
```
manifesto -> principle

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+* Check on text layout, alignment and wrapping. Box size should be adjustable to fit all the content
+
+### [mozorg/contribute/signup.lang](https://www.mozilla.org/contribute/signup/)
+There are different ways to access the signup dialog box above the footer area.  This is from the Contribute page.  
```
No double spaces after a period

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+
+### [mozorg/contribute/signup.lang](https://www.mozilla.org/contribute/signup/)
+There are different ways to access the signup dialog box above the footer area.  This is from the Contribute page.  
+* Click on any of the six icons or the links below each of them, you will be prompt to the Sign Up section
```
prompt -> prompted

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -65,3 +65,55 @@ In some cases pages receive major updates that require a complete rewrite of the
+### [mozorg/contribute/signup.lang](https://www.mozilla.org/contribute/signup/)
+There are different ways to access the signup dialog box above the footer area.  This is from the Contribute page.  
+* Click on any of the six icons or the links below each of them, you will be prompt to the Sign Up section
+* Leave the name field empty, click on the **Sign Up Now** button. A text bubble will appear
```
Note that this is the sign-up for the newsletter, not for the contribute page. That's available even at the bottom of the home page.

The error bubbles come from Firefox, e.g. try signing up at the bottom of the Italian page, the warning will be in English https://www.mozilla.org/it/

Personally I would drop this section

peiying2 committed:
Review feedback implementation

- Need info for sendto.lang locales on production
- Currently, half of the Sync.lang contains is for the static page.  Nothing to address older FF or older sync, etc.  
- Let's wait for Theo's input on whether to keep the "signup" section.  
- Any other pages are not covered here?

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -36,32 +36,87 @@ It’s highly advised you to ask other community members to conduct peer review
+
+### [mozorg/about/history.lang](https://www.mozilla.org/about/history/)
+* The slides should automatically scroll to the next in about 10 seconds or so 
+* If it doesn't or it stops, click the next or previous arrows to go to the next slide
```
Looks like there's a rogue straight quote instead of an apostrophe

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -36,32 +36,87 @@ It’s highly advised you to ask other community members to conduct peer review
+* Check on text layout, alignment and wrapping. Box size should be adjustable to fit all the content
+
+### [mozorg/contribute/signup.lang](https://www.mozilla.org/)
+There are different ways to access the signup dialog box above the footer area. This is for signing up newsletters. 
```
I think it should be "signing up to newsletters"

peiying2 committed:
Implementations #2

- adding production locales for 'sendto.lang'
- fix apostrophe 
- grammar, :-P!
- minor formatting change

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -36,32 +36,86 @@ It’s highly advised you to ask other community members to conduct peer review
+
+### [firefox/new/horizon.lang](https://www.mozilla.org/firefox/new/)
+* Once page is loaded, click on the link called **Download Firefox for another platform**
+* For alternative message, change URL to [Scene2](https://www.mozilla.org/en-US/firefox/new/?scene=2)
```
Remove en-US from the link

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -36,32 +36,86 @@ It’s highly advised you to ask other community members to conduct peer review
+### firefox/tracking-protection-tour.lang
+* Open Firefox and start a New Private Window
+* Once the window is launched, click on **See how it works** button
+* You are taken to [step one](https://www.mozilla.org/en-US/firefox/51.0.1/tracking-protection/start/?step=1)
```
Remove en-US from all these links

peiying2 committed:
removal of en-US in the links

flodolo commented on products/mozilla_org/test_mozilla_org.md:
```diff
@@ -36,32 +36,86 @@ It’s highly advised you to ask other community members to conduct peer review
 * Font support and readability
 * Footer: verify that the translation of the link is coherent and the link is functional
-* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/en-US/ to confirm
+* Language list: Is your language listed as one of the options? Check https://www-dev.allizom.org/ to confirm
```
Commenting again here, since doing it on a commit is confusing: this one actually needs to point to /en-US

peiying2 committed:
add en-US back in line #43

peiying2 committed:
remove sync and signup sections

Release what is good to go and decide on the pending sections.
